h5c/vic/src/vic-webapp/src/app/create-vch-wizard/compute-capacity/compute-capacity.html

```diff
@@ -412,5 +412,64 @@
       <span>MB</span>
 
     </div>
+
+    <!-- VM-Host Affinity -->
+    <ng-container *ngIf="selectedResourceIsCluster">
+      <div class="form-group row">
+        <div class="col-xs-4">
+          <label for="vm-host-affinity" class="affinity-label">
+            VM-Host Affinity
```

Comment:
we should start getting into the habit of putting these labels text into the locales file (we only support en_US currently).. src/main/webapp/locales/